public/antigravity.css

```
@@ -689,15 +793,130 @@ body.antigravity-theme .settings-toggle[aria-checked="true"]::after {
+}
+
+html[data-theme="dark"] body.antigravity-theme .desk-eyebrow {
+  background: rgba(158, 139, 255, 0.12);
+  border-color: rgba(158, 139, 255, 0.24);
+}
+
+body.antigravity-theme .desk-eyebrow-dot {
+  width: 5px;
+  height: 5px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)